MDC-111/ObjectiveC/Complete/MDC-111/ViewController.m

```diff
@@ -94,6 +95,8 @@ - (IBAction)saveDidTouch:(id)sender {
   }]];
   [saveAlert addAction:[MDCAlertAction actionWithTitle:@"Cancel" handler:^(MDCAlertAction * _Nonnull action) {
   }]];
+  MDCAlertScheme *alertScheme = [[MDCAlertScheme alloc] init];
```

Comment:
in this case, where was the removal of applying the semantic and typography schemes like in the swift case?

Reply:
I'm not sure I understand. This is the PR to *add* any themers. I don't know why the Swift code had any previously, but it certainly should not have.  I suspect [the Alert themer may have been added by accident](https://github.com/material-components/material-components-ios-codelabs/commit/69c3dd2bcf9ab1cdf07942d330e64694110baecf) because the commit message only mentions Buttons.